src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/osd-data.pipe.ts

```diff
@@ -0,0 +1,57 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+import _ from 'lodash';
+
+@Pipe({
+  name: 'osdDataPipe'
+})
```

Comment:
In general I'm all in favor of pipes, but when they're reusable. This pipe can only be used in a single place for a single type of input, so it's extremely specific :/

Reply:
Thanks @epuertat, No need of the pipe now, so removed it.